packages/app-project/src/shared/components/StandardLayout/StandardLayout.spec.js

```diff
@@ -182,47 +183,96 @@ describe('Component > StandardLayout', function () {
   })
 
   describe('in admin mode', function () {
-    before(function () {
-      const snapshot = {
-        project: {
-          configuration: {
-            languages: ['en']
-          },
-          slug: 'Foo/Bar',
-          strings: {
-            display_name: 'Foobar'
+    describe('when the user is loading', function () {
+      before(function () {
+        const snapshot = {
+          project: {
+            configuration: {
+              languages: ['en']
+            },
+            slug: 'Foo/Bar',
+            strings: {
+              display_name: 'Foobar'
+            },
+            links: {
+              active_workflows: ['1']
+            }
           },
-          links: {
-            active_workflows: ['1']
+          user: {
+            loadingState: asyncStates.loading
           }
-        },
-        user: {
-          id: '1',
-          admin: true,
-          login: 'zooVolunteer'
         }
-      }
-      window.localStorage.setItem('adminFlag', true)
-      render(<StandardLayout />, { wrapper: withStore(snapshot)})
-      projectPage = screen.getByTestId('project-page')
-      zooHeader = screen.getByRole('banner')
-      zooFooter = screen.getByRole('contentinfo')
-      adminToggle = within(zooFooter).getByRole('checkbox', { name: 'Admin Mode' })
-    })
+        window.localStorage.setItem('adminFlag', true)
+        render(<StandardLayout />, { wrapper: withStore(snapshot)})
+        projectPage = screen.getByTestId('project-page')
+        zooHeader = screen.getByRole('banner')
+        zooFooter = screen.getByRole('contentinfo')
+        adminToggle = within(zooFooter).queryByRole('checkbox', { name: 'Admin Mode' })
+      })
 
-    after(function () {
-      window.localStorage.removeItem('adminFlag')
-    })
+      after(function () {
+        window.localStorage.removeItem('adminFlag')
+      })
 
-    it('should have a striped border', function () {
-      expect(projectPage).to.be.ok()
-      const borderImage = window.getComputedStyle(projectPage)['border-image']
-      expect(borderImage).to.equal(adminBorderImage)
+      it('should not have a striped border', function () {
+        expect(projectPage).to.be.ok()
+        const borderImage = window.getComputedStyle(projectPage)['border-image']
+        expect(borderImage).to.equal('')
+      })
+
+      it('should not show the admin toggle in the footer', function () {
+        expect(adminToggle).to.be.null()
+      })
+
+      it('should preserve the stored adminFlag', function () {
+        expect(window.localStorage.getItem('adminFlag')).to.equal('true')
+      })
```

Comment:
This test passes on this branch but fails on the main branch. I think it reproduces the bug where admin mode can be accidentally turned off by removing the stored `adminFlag`.